client/homebrew/pages/vaultPage/vaultPage.jsx

```diff
@@ -0,0 +1,473 @@
+                </Nav.item>
+            </Nav.section>
+            <Nav.section>
+                <NewBrew />
+                <HelpNavItem />
+                <RecentNavItem />
+                <Account />
+            </Nav.section>
+        </Navbar>
+    );
+
+    const validateForm = () => {
+        //form validity: title or author must be written, and at least one renderer set
+        const { current: titleInput } = titleRef;
+        const { current: legacyCheckbox } = legacyRef;
+        const { current: v3Checkbox } = v3Ref;
+        const { current: authorInput } = authorRef;
+
+        const isTitleValid = titleInput.validity.valid && titleInput.value;
+        //because a pattern attr is set in the input, title must be over 2 chars long
+        const isAuthorValid = authorInput.validity.valid && authorInput.value;
+        const isCheckboxChecked = legacyCheckbox.checked || v3Checkbox.checked;
+
+        const isFormValid =
+            (isTitleValid || isAuthorValid) && isCheckboxChecked;
+
+        return isFormValid;
+    };
+
+    const disableSubmitIfFormInvalid = () => {
+        const { current: submitButton } = searchButtonRef;
+        submitButton.disabled = !validateForm();
+    };
+
+    const renderForm = () => (
+        <div className="brewLookup">
+            <h2 className="formTitle">Brew Lookup</h2>
+            <div className="formContents">
+                <label>
+                    Title of the brew
+                    <input
+                        ref={titleRef}
+                        type="text"
+                        name="title"
+                        defaultValue={title}
+                        onKeyUp={disableSubmitIfFormInvalid}
+                        pattern=".{3,}"
+                        title="At least 3 characters"
+                        onKeyDown={(e) => {
+                            if (e.key === 'Enter') {
+                                if (!searchButtonRef.current.disabled) {
+                                    loadPage(1, true, true);
+                                }
+                            }
+                        }}
+                        placeholder="v3 Reference Document"
+                    />
+                </label>
+
+                <label>
+                    Author of the brew
+                    <input
+                        ref={authorRef}
+                        type="text"
+                        name="author"
+                        pattern=".{1,}"
+                        defaultValue={author}
+                        onKeyUp={disableSubmitIfFormInvalid}
+                        onKeyDown={(e) => {
+                            if (e.key === 'Enter') {
+                                if (!searchButtonRef.current.disabled) {
+                                    loadPage(1, true, true);
+                                }
+                            }
+                        }}
+                        placeholder="Gazook89"
```

Comment:
Of course authors are case sensitive, the only case insensitive here is the title, and because we have is set that way.

Reply:
Author values in the database are case sensitive.

We should treat the user's search criteria as case insensitive though.

Reply:
Can you use a different placeholder other than my account name-- perhaps just `homebrewery`?

Or, i suggest no placeholders at all.  I don't think there is much gained by them in this case (the required input isn't complicated).  And they styling of placeholders isn't sufficiently different enough from normal text in the input, so there is a chance a user may think the fields are already filled (the gray is fairly similar to black text).

Reply:
> Author values in the database are case sensitive.
> 
> We should treat the user's search criteria as case insensitive though.

Been looking into that, because it is not a text search we should have a [case insensitive index for this to work](https://www.mongodb.com/docs/manual/core/index-case-insensitive/). Not sure that is the way to go. Maybe for a later PR.